pkg/config/load.go

```
@@ -0,0 +1,227 @@
+	v.AddConfigPath(path)
+	v.SetConfigName(fileName)
+	err := v.MergeInConfig()
+	if err != nil {
+		return nil
+	}
+
+	return nil
+}
+
+// applyDefaultConfiguration apply default configuration for the atmos config.
+func applyDefaultConfiguration(v *viper.Viper) error {
```

golangci-lint (1.62.2)

func `applyDefaultConfiguration` is unused

(unused)